ddl/reorg_test.go

```diff
@@ -65,7 +65,7 @@ func (s *testDDLSuite) TestReorg(c *C) {
 	handle := int64(100)
 	f := func() error {
 		d.reorgCtx.setRowCountAndHandle(rowCount, handle)
-		time.Sleep(20 * testLease)
+		time.Sleep(1*ReorgWaitTimeout + 100*time.Millisecond)
```

Comment:
Could we modify ReorgWaitTimeout's value to a smaller value to make test faster?

Reply:
Set it in testDDLSuite.SetupSuit().